src/main/java/ch/njol/skript/expressions/ExprCommand.java

```diff
@@ -72,23 +71,25 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 	
 	@Override
 	@Nullable
-	protected String[] get(final Event e) {
-		final String s;
+	protected String[] get(Event event) {
+		final String command;
 
-		if (e instanceof PlayerCommandPreprocessEvent) {
-			s = ((PlayerCommandPreprocessEvent) e).getMessage().substring(1).trim();
-		} else if (e instanceof ServerCommandEvent) {
-			s = ((ServerCommandEvent) e).getCommand().trim();
+		if (event instanceof PlayerCommandPreprocessEvent) {
+			command = ((PlayerCommandPreprocessEvent) event).getMessage().substring(1).trim();
+		} else if (event instanceof ServerCommandEvent) {
+			command = ((ServerCommandEvent) event).getCommand().trim();
+		} else if (event instanceof UnknownCommandEvent) {
```

Comment:
This is a Paper exclusive event. Needs a Skript.classExists method check or reference to private static final boolean as said in my other comment.